articles/custom-error-pages.md

```diff
@@ -0,0 +1,113 @@
+# Customizing Error Pages
+
+In the event of an authorization error, you may choose to display to your users either the default Auth0 error page or a customized error page.
+
+This document begins by covering the configuration options available to you via the Management Portal and ends with information on performing the same actions by making the appropriate calls to the Management API.
+
+## Customizing Error Pages via the Management Portal
+
+To get to the error page settings:
+
+1. Go to the top right-hand side of the Auth0 Management Portal
+2. Click on your user name/icon
+3. Choose "Account Settings" in the menu the pops open.
+
+![](/media/articles/error-pages/account-settings.png)
+
+### The Auth0 Default Error Page
+
+You can choose to display to your users the default Auth0 error page. This page can be minimally customized with the following pieces of information (all fields optional):
+
+-	Friendly Name: a user-friendly version of your company's name;
+-	Logo URL: the path to the logo you want to show to users;
+-	Support Email: the email address for your Support team;
+-	Support URL: the URL of your Support team's webpage.
+
+![](/media/articles/error-pages/error-page-settings.png)
+
+### Customized Error Pages
+
+If you choose to display a custom error page, you have two options:
+
+-	You may redirect the user to a custom error page;
+-	You may configure Auth0 to render a custom error page on your behalf (please note that this feature is only available via the Management API).
+
+#### Redirecting Users to a Custom Error Page
+
+To redirect users to a custom error page:
+
+1. On the Account Settings page, scroll down to the Error Pages section.
+2. Select the option "redirect users to your own error page."
+3. Provide the URL of the error page you would like your users to see.
+
+![](/media/articles/error-pages/redirect-error-page.png)
+
+## Customizing Error Pages via the Management API
+Instead of using the Management Portal, you may configure your error pages by making the appropriate PATCH call to the Management API.
+
+![](/media/articles/error-pages/patch-tenant-settings.png)
+
+Prior to beginning, please ensure that you are logged in to an account that is permitted to make changes to your Auth0 configuration.
+
+1. Navigate to the [APIv2 Explorer Page](/api/v2).
```

Comment:
So for this, i would say we shouldnt recommend using the API explorer as a UI interface. I would document this explaining how to use the API directly. Explaining the PATCH call to the api and what the request will look like. In the beginning of the section maybe mention that you can either use the API directly or if they want they can try this via the management explorer.

I would show how to send the command using CURL. That is kind of the standard way to do it.

Bonus points if you want to create a run in postman button: http://blog.getpostman.com/2016/01/28/the-run-in-postman-button-for-improved-documentation/